src/buildCommon.js

```diff
@@ -636,8 +645,8 @@ const retrieveBaseFontName = function(font: string): string {
         case "texttt":
             baseFontName = "Typewriter";
             break;
-        default:
-            throw new Error(`Invalid font provided: ${font}`);
+        default: // probably a font added by a plugin
+            baseFontName = font;
```

Comment:
The comment should probably be more definitive, e.g.
```
default:
   baseFontName = font;  // use by fonts added by a plugin
```
I like that things line up in your plugin.  I wish we didn't have so many mappings for font related things that we have to maintain.

Reply:
Done